tests/net/virtual/testcase.yaml

```diff
@@ -0,0 +1,25 @@
+common:
+  depends_on: netif
+  min_ram: 32
+  tags: net tunnel virtual
```

Comment:
please add integration_platforms and list the platforms where this should run on in CI for pull requests.